src/main/webapp/app/exam/participate/exam-cover/exam-participation-cover.component.ts

```diff
@@ -75,16 +91,54 @@ export class ExamParticipationCoverComponent implements OnInit, OnDestroy {
     /**
      * check, whether exam has started yet and we therefore can enable the Start Exam Button
      */
-    enableStartButton() {
-        if (this.confirmed && this.exam && this.exam.startDate && moment(this.exam.startDate).isBefore(moment())) {
+    async enableStartButton() {
+        let fullname = '';
+        await this.accountService.identity().then((user) => {
+            fullname = user?.name ?? '';
+        });
+
+        if (this.fullname && this.fullname !== fullname) {
+            this.falseName = true;
+        } else {
+            this.falseName = false;
+        }
+
+        if (this.fullname && this.fullname === fullname && this.confirmed && this.exam && this.exam.startDate && moment(this.exam.startDate).isBefore(moment())) {
             return true;
         } else {
             return false;
         }
     }
 
     /**
-     * TODO: add session management, this function is bound to the start exam button
+     * Starts the exam and creates an exam session and stores the token in the browser
+     */
+    startExam() {
+        this.examSessionService.createExamSession(this.courseId, this.examId).subscribe((response) => {
```

Comment:
In this case, the (probably non existing) REST Call "start-exam" should be invoked, there should be no separated call to create an exam session